src/sass/partials/components/_nav.scss

```diff
@@ -41,3 +41,47 @@
     }
   }
 }
+
+.breadcrumb {
+
+  &--list {
+    background-color: transparent;
+    display: flex;
+    margin-top: 0;
+    transition: .15s transform ease-out;
```

Comment:
Your right, my mistake. I thought I saw that somewhere but looks like I'm wrong.